src/hooks/useOnboardingCourse.tsx

```diff
@@ -24,32 +21,29 @@ export const OnboardingCourseContextProvider = ({
 }: {
   children: React.ReactNode;
 }) => {
-  const {
-    data,
-    isLoading: isAppConfigLoading,
-    isFetched: isAppConfigFetched,
-    error,
-  } = useAppConfig();
+  const { activeSubject, isLoading } = useActiveProject();
+  const data = activeSubject?.project.appConfig;
+
   const onboardingCourseUrl = data?.onboardingCourse?.url;
   const onboardingCourseTitle = data?.onboardingCourse?.title;
-  const { activeProject } = useActiveProject();
   const [storedDidLaunchResult, storeDidLaunch, isStorageLoaded] =
     useAsyncStorage(
-      `${activeProject?.id}-didLaunchOnboardingCourse`,
-      !!activeProject?.id,
+      `${activeSubject?.project?.id}-didLaunchOnboardingCourse`,
+      !!activeSubject?.project?.id,
     );
+
   const [didLaunchCourse, setDidLaunchCourse] = useState<boolean | undefined>(
     undefined,
   );
 
-  const isLoading = isAppConfigLoading || !isStorageLoaded;
-  const isFetched = isAppConfigFetched && isStorageLoaded;
+  const isOnboardingLoading = isLoading || !isStorageLoaded;
+  const isFetched = isStorageLoaded;
```

Comment:
Should this be
```suggestion
  const isFetched = !!data && isStorageLoaded;
```

Reply:
My actual intention was to drop the isFetched flag and just use the isLoading flag instead. I will review and update accordingly.